config/helm/aws-node-termination-handler/templates/daemonset.linux.yaml

```diff
@@ -154,10 +154,10 @@ spec:
       nodeSelector:
         {{ include "aws-node-termination-handler.nodeSelectorTermsOs" . }}: linux
         {{- with .Values.nodeSelector }}
-          {{- . | nindent 8 }}
+          {{- toYaml . | nindent 8 }}
```

Comment:
What was it converting the value to previously?

Reply:
It just directly inserted the string.